package.json

```diff
@@ -10,6 +10,9 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fullcalendar/core": "^6.1.11",
```

Comment:
do you need this dependency? I don't see it used throughout the app. I only see fullcalendar/react and fullcalendar/daygrid being used. I don't know for sure..just curious

Reply:
Removed unused dependency